bedevere/bpo.py

```diff
@@ -83,10 +83,67 @@
         await set_status(event, gh)
 
 
+@router.register("issue_comment", action="edited")
+@router.register("issue_comment", action="created")
+async def hyperlink_bpo_text(event, gh, *args, **kwargs):
+    body = event.data["comment"]["body"]
+    new_body = create_hyperlink_in_comment_body(body)
+    if new_body != body:
+        body_data = {"body": new_body, "maintainer_can_modify": True}
+        await gh.patch(event.data["comment"]["url"], data=body_data)
+
+
 def create_success_status(found_issue):
     """Create a success status for when an issue number was found in the title."""
     issue_number = found_issue.group("issue")
     url = f"https://bugs.python.org/issue{issue_number}"
     return util.create_status(STATUS_CONTEXT, util.StatusState.SUCCESS,
                               description=f"Issue number {issue_number} found",
                               target_url=url)
+
+
+def check_hyperlink(match):
+    issue = match.groupdict()['issue']
+    markdown_link_re = re.compile(r"""
+                                    \[\s*bpo-(?P<issue>{issue})\s*\]   
+                                    \(\s*https://www.bugs.python.org/issue{issue}\s*\)""".format(issue=issue),
+                                    re.VERBOSE)
+    html_link_re = re.compile(r""" <a
+                                   \s*href\s*=\s*[",']\s*
+                                   https://www.bugs.python.org/issue{issue}
+                                   \s*[",']\s*>
+                                   \s*bpo-(?P<issue>{issue})\s*
+                                   </a>""".format(issue=issue),
+                                   re.VERBOSE)
+    # The following approach takes care of cases like bpo-123 [bpo-123]...
```

Comment:
Maybe the comments here and in line 126 should be part of docstring instead of inline codes.